### DISCUSSION TITLE
Notification Mails not being sent

### DISCUSSION THREAD

**Discussion by michael-schramm** [Leantime/leantime#1738] [Q&A]
Hi,
I am using Leantime 2.3.25.
Email notifications is not working.
But when I request a password reset, I get an email.
I configured email via SMTP.

Any idea why we don't get notification mails at all?

Thanks!

**Comment by marcelfolaron** [comment]
Any errors in you logs/error.log file?
You can try to set debug to 1 and watch the smtp output in the log file to
determine where it fails.

**Reply by michael-schramm** [reply]
Hi,
where can I set debug to 1?

Thanks!
:-)

**Reply by michael-schramm** [reply]
Oh!
I found it on the sample configuration file, but It wasn't in my configuration.php file ??!!
That's very strange.
I just set debug to 1 and will see what happens.
In the meantime I also updated to 2.4 beta.

**Comment by michael-schramm** [comment]
It seems that the cron job ist not running, but I don't know why.

**Reply by ema-vit** [reply]
Hi
I have the same problem it seems. I wrote some code to test and I think there is a misconfiguration.
You solve?
Thanks

**Reply by ema-vit** [reply]
I'm sorry . this is my debug erro file : 
4 invalid address (to) : 1                      [1 I discover is id user, I think the system not replace id with email]

**Comment by ema-vit** [comment]
I have the same problem it seems. I wrote some code to test and I think there is a misconfiguration.
Can I solve?

This is my debug error file :
4 invalid address (to) : 1 [1 I discover is id user, I think the system not replace id with email]

**Comment by gloriafolaron** [comment]
Are these still an issue for you both? 

Will add -- support can be quicker via Discord as well if you're not in there yet. https://discord.gg/sugXFb79Hn
